src/Components/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL/AspireEFPostgreSqlExtensions.cs

```diff
@@ -64,14 +94,14 @@ public static partial class AspireEFPostgreSqlExtensions
 
         configureSettings?.Invoke(settings);
 
-        builder.Services.AddDbContextPool<TContext>(ConfigureDbContext);
+        builder.Services.AddDbContextPool<TContextService, TContextImplementation>(ConfigureDbContext);
```

Comment:
Should pooling be used by default ?

Reply:
Good question but I would defer to the OG author. This PR does not alter or introduce that behavior.

Reply:
Thanks for that info.
I just wondered, because the normal EF AddDbContext of course is a non-pooled setup.

The previous PR could be introducing certain "problems"

Reply:
@jbroumels if I had to pick I'd prefer restoring the previous functionality where this is configurable. In our own projects, over time, we've adopted a lot of opinionated code very similar to Aspire, where we take safe, sane, defaults and the rest is taken from `appsettings.json` (including pooling).

Reply:
Agree, the defaults should always be the safe manner.
I overlooked it didn't came from this PR directly, so it doesn't belong here.

Will look into the referenced commit and PR.

Closing this comment section here for now :)

Reply:
You may file an issue about why pooling or not should be the default, even for consistency in the APIs, I would loop some EF people too.